### DISCUSSION TITLE
[Request] Docs

### DISCUSSION THREAD

**Discussion by ikilobyte** [mscharley/dot#400] [Ideas]
### Is your feature request related to a problem? Please describe it.

https://mscharley.github.io/dot/docs/ It's too simple. Is there a sample document?

**Comment by mscharley** [comment]
Hi! You're right that there's definitely some pretty big gaps in documentation in this project at the moment. For the sake of being as helpful as possible as quickly as possible though, is there anything in particular that you're struggling with?

In terms of examples, there's [the example on the main page](https://mscharley.github.io/dot/docs/dot/_root/#example). This is a very simple 'getting started' style example though.
